src/design/Basics.js

```diff
@@ -135,6 +135,37 @@ var Basics = React.createClass({
     <span className="swatch graph-6"></span>
 
   </section>
+      
+  <section>
+    <a className="anchor" id="padding"></a>
+    <h2>Padding</h2>
+      
+      <p>Padding on elements and in mixins is based off of the inuitcss <code>$inuit-base-spacing-unit</code> variable, which is set to 24px. Hardcoded pixel values for padding should rarely be used; use spacing variables instead.</p>
```

Comment:
What "mixins" are being referred to here? I think we have removed the need for any mixins.